api/tests/app/test_camera.py

```diff
@@ -0,0 +1,645 @@
+        create_a_camera(client, camera_sample)
+
+        body = {
+            "pts_destination": [
+                [
+                    0,
+                    0
+                ],
+                [
+                    0,
+                    0
+                ],
+                [
+                    0,
+                    0
+                ],
+                [
+                    0,
+                    0
+                ]
+            ]
+        }
+
+        camera_id = camera_sample['id']
+
+        with pytest.raises(numpy.linalg.LinAlgError):
+            client.post(f'/cameras/{camera_id}/homography_matrix', json=body)
+
+    def test_try_set_coordinates_non_existent_id(self, config_rollback, pts_destination):
+        client, config_sample_path = config_rollback
+
+        camera_id = 'Non-existent ID'
+        body = pts_destination
+        response = client.post(f'/cameras/{camera_id}/homography_matrix', json=body)
+
+        assert response.status_code == 404
+        assert response.json() == {'detail': f'The camera: {camera_id} does not exist'}
+
+    def test_try_set_coordinates_empty_request_body(self, config_rollback, camera_sample,
+                                                    rollback_screenshot_camera_folder):
+        client, config_sample_path = config_rollback
+        create_a_camera(client, camera_sample)
+
+        camera_id = camera_sample['id']
+        body = {}
+        response = client.post(f'/cameras/{camera_id}/homography_matrix', json=body)
+
+        assert response.status_code == 400
+        assert response.json() == {'detail': [{'loc': ['body', 'pts_destination'], 'msg': 'field required',
+                                               'type': 'value_error.missing'}], 'body': {}}
+
+    def test_try_set_coordinates_bad_request_body(self, config_rollback, camera_sample,
+                                                  rollback_screenshot_camera_folder):
+        client, config_sample_path = config_rollback
+        create_a_camera(client, camera_sample)
+
+        camera_id = camera_sample['id']
+        body = {'pts_destination': [None]}
+        response = client.post(f'/cameras/{camera_id}/homography_matrix', json=body)
+
+        assert response.status_code == 400
+        assert response.json() == {'detail': [{'loc': ['body', 'pts_destination'], 'msg': 'ensure this value has at '
+                                                                                          'least 4 items',
+                                               'type': 'value_error.list.min_items', 'ctx': {'limit_value': 4}}],
+                                   'body': {'pts_destination': [None]}}
+
+
+# pytest -v api/tests/app/test_camera.py::TestClassGetCameraCalibrationImage
+class TestClassGetCameraCalibrationImage:
+    """ Get Camera Calibration Image, GET /cameras/{camera_id}/calibration_image """
+    """ TODO: Maybe, here we could go deeper into the response.json() """
```

Comment:
Do you mean when validating error messages? I think you did it in a test of this class

Reply:
Yes, the comments refer to the first test only.

Reply:
I'd either compare the successful  json to what's expected or just remove the comment

Reply:
Okey, changed!